xml/System/Environment.xml

```diff
@@ -1041,26 +1041,29 @@ The following example creates environment variables for the <xref:System.Environ
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- The names and values for the environment variables are stored as key-value pairs in the returned <xref:System.Collections.IDictionary>.
+
+The names and values for the environment variables are stored as key-value pairs in the returned <xref:System.Collections.IDictionary>.
 
 ### On Windows systems
 
 On Windows systems, the `GetEnvironmentVariables` method returns the following environment variables:
 
 - All per-machine environment variables that are defined at the time the process is created, along with their values.
-
 - All per-user environment variables that are defined at the time the process is created, along with their values.
-
 - Any variables inherited from the parent process from which the .NET application was launched or added to the process block while the process is running. Environment variables are added while the process is running by calling either the <xref:System.Environment.SetEnvironmentVariable%28System.String%2CSystem.String%29> method or the <xref:System.Environment.SetEnvironmentVariable%28System.String%2CSystem.String%2CSystem.EnvironmentVariableTarget%29> method with a `target` value of <xref:System.EnvironmentVariableTarget.Process?displayProperty=nameWithType>.
 
-### On non-Windows systems
+### On Unix-like systems
 
-On non-Windows systems, the `GetEnvironmentVariables` method retrieves the name and value of all environment variables that are inherited from the parent process that launched the `dotnet` process or that are defined within the scope of the `dotnet` process itself. Once the `dotnet` process ends, these latter environment variables cease to exist.
+On Unix-like systems, the `GetEnvironmentVariables` method retrieves the name and value of all environment variables that are inherited from the parent process that launched the `dotnet` process or that are defined within the scope of the `dotnet` process itself. Once the `dotnet` process ends, these latter environment variables cease to exist.
```

Comment:
I have the same question as the other PR: Is it worth mentioning that macOS is "unix-like"?